sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/utils/crypto/CryptoNoOpHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.azure.mobile.utils.crypto;
+
+import android.content.Context;
+
+import java.security.KeyStore;
+
+/**
+ * Handler that does not actually encrypt anything.
+ */
+class CryptoNoOpHandler implements CryptoHandler {
```

Comment:
Instead of having this class, why don't you replace CryptoHandler interface by class as a base class? Just my opinion.